undetected_chromedriver/webelement.py

```diff
@@ -1,7 +1,28 @@
+from selenium.webdriver.common.by import By
 import selenium.webdriver.remote.webelement
 
 
 class WebElement(selenium.webdriver.remote.webelement.WebElement):
+    def click_safe(self):
+        super().click()
+        self._parent.reconnect(0.1)
+
+    def children(
+        self, tag=None, recursive=False
+    ) -> list[selenium.webdriver.remote.webelement.WebElement]:
+        """
+        returns direct child elements of current element
+        :param tag: str,  if supplied, returns <tag> nodes only
+        """
+        script = "return [... arguments[0].children]"
+        if tag:
+            script += ".filter( node => node.tagName === '%s')" % tag.upper()
+        if recursive:
+            return _recursive_children(self, tag)
+        return self._parent.execute_script(script, self)
```

Comment:
Seems pretty expensive to call `execute_script` for every child of an element. Is it not easier to utilise js `getElementsByTagName`.

Reply:
sure. And how do you think selenium get's their elements. all javascript calls....non issue